wormhole-connect/src/views/v2/Bridge/Routes/SingleRoute.tsx

```diff
@@ -0,0 +1,318 @@
+    sourceToken,
+    destToken,
+    amount,
+    route,
+    toNativeToken,
+  });
+
+  const totalFees = useMemo(() => {
+    let total = 0;
+
+    if (relayerFee) {
+      total += relayerFee || 0;
+    }
+
+    if (props.showDestinationGasFee && receiveNativeAmt) {
+      total += receiveNativeAmt || 0;
+    }
+
+    return (
+      <>
+        <Typography color={theme.palette.text.secondary} fontSize={14}>
+          Total fees
+        </Typography>
+        {isFetchingFees || isFetchingQuote ? (
+          <CircularProgress size={14} />
+        ) : (
+          <Typography fontSize={14}>
+            {Number.parseFloat(toFixedDecimals(`${total}`, 6))}
+          </Typography>
+        )}
+      </>
+    );
+  }, [
+    isFetchingFees,
+    isFetchingQuote,
+    props.showDestinationGasFee,
+    relayerFee,
+    receiveNativeAmt,
+  ]);
+
+  const bridgeFee = useMemo(
+    () => (
+      <Stack direction="row" justifyContent="space-between">
+        <Typography
+          color={theme.palette.text.secondary}
+          fontSize={14}
+          paddingLeft="8px"
+        >
+          Bridge fee
+        </Typography>
+        {isFetchingQuote ? (
+          <CircularProgress size={14} />
+        ) : (
+          <Typography fontSize={14}>{relayerFee}</Typography>
+        )}
+      </Stack>
+    ),
+    [isFetchingQuote, relayerFee],
+  );
+
+  const destinationGasFee = useMemo(() => {
+    if (!receiveNativeAmt) {
+      return null;
+    }
+
+    return (
+      <Stack direction="row" justifyContent="space-between">
+        <Typography
+          color={theme.palette.text.secondary}
+          fontSize={14}
+          paddingLeft="8px"
+        >
+          Destination gas fee
+        </Typography>
+        {isFetchingQuote ? (
+          <CircularProgress size={14} />
+        ) : (
+          <Typography fontSize={14}>
+            {Number.parseFloat(toFixedDecimals(`${receiveNativeAmt}`, 6))}
+          </Typography>
+        )}
+      </Stack>
+    );
+  }, [isFetchingQuote, receiveNativeAmt]);
+
+  const feesBreakdown = useMemo(() => {
+    return (
+      <>
+        {bridgeFee}
+        {props.showDestinationGasFee && destinationGasFee}
+      </>
+    );
+  }, [bridgeFee, destinationGasFee, props.showDestinationGasFee]);
+
+  const timeToDestination = useMemo(() => {
+    return (
+      <>
+        <Typography color={theme.palette.text.secondary} fontSize={14}>
+          Time to destination
+        </Typography>
+
+        {estimatedTime ? (
+          <Typography fontSize={14}>{estimatedTime}</Typography>
+        ) : (
+          <CircularProgress size={14} />
+        )}
+      </>
+    );
+  }, [estimatedTime]);
+
+  const showWarning = useMemo(() => {
+    if (!props.config.route) {
+      return false;
+    }
+
+    const routeConfig = RouteOperator.getRoute(props.config.route);
+
+    return !(
+      routeConfig.AUTOMATIC_DEPOSIT ||
+      (destChain && isGatewayChain(destChain)) ||
+      destChain === 'sei' ||
```

Comment:
I want to stop having route- or chain-specific logic in our view code. Connect UI should be "dumb" and SDK's abstractions should be enough to satisfy UI needs. what would it take to get rid of these?

Reply:
This was a copy/paste from the old code to show/hide the warning "This transfer requires two transactions ..."
@artursapek is there a utility function or other means to check without a specific chain? Or would `!routeConfig.AUTOMATIC_DEPOSIT` be sufficient?

Reply:
I think for now we can just change this to `!routeConfig.AUTOMATIC_DEPOSIT`. We are not including sei or cosmos gateway initially and whether those are automatic or manual should be provided by the SDK